x-pack/solutions/search/plugins/search_synonyms/public/components/connect_to_api/connect_to_api_flyout.tsx

```diff
@@ -78,11 +79,41 @@ export const ConnectToApiFlyout: React.FC<ConnectToApiFlyoutProps> = ({ onClose,
           </p>
         </EuiText>
         <EuiSpacer />
-        <EuiThemeProvider colorMode="dark">
-          <EuiCodeBlock language="json" isCopyable fontSize="m">
-            {getExampleCode(rulesetId)}
-          </EuiCodeBlock>
-        </EuiThemeProvider>
+        <EuiAccordion
+          id="connectToIndexNewIndexAccordion"
+          buttonContent={i18n.translate(
```

Comment:
I think we can make these accordion labels headings.  This might not be quite right, but for example:

```
<EuiTitle size="xs">
<h3>{...button content...}</h3
</EuiTitle>
```